### PR TITLE
68 registration and deletion of users

### LINKED ISSUE #68: Registration and Deletion of Users
Models-part of https://github.com/ontohub/ontohub-backend/issues/86

### PR DESCRIPTION
**Pull request by eugenk**
This shall fix #68.

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -2,12 +2,21 @@
 
 Devise.setup do |config|
   # ==> Configuration for :database_authenticatable
-  # For bcrypt, this is the cost for hashing the password and defaults to 15. If
+  # For bcrypt, this is the cost for hashing the password and defaults to 11. If
```
Comment (phyrog) [inline config/initializers/devise.rb]: Shouldn't this be 13, as in line 12?

Comment (eugenk) [inline config/initializers/devise.rb]: No, it was really 11. I increased it to 13 in the config. I don't know why I put the 15 there in the first place...

See https://github.com/plataformatec/devise/blob/f7b6d786066cef2f5e8d2ce9c6b6cc83918580eb/lib/generators/templates/devise.rb#L100

File: db/migrate/20160902111740_initial_schema.rb [hunk 2]
```diff
@@ -11,8 +11,8 @@
 
       column :url_path, String, unique: true
 
-      column :name, String
       column :slug, String, unique: true
+      column :real_name, String, null: true
```
Comment (phyrog) [inline db/migrate/20160902111740_initial_schema.rb]: What do you think about renaming this column to `display_name`, as some people might not want to use their real name (so this column would not always include the real name)?

Comment (eugenk) [inline db/migrate/20160902111740_initial_schema.rb]: We could do this. Shall I do it here or in another pull request?

File: config/initializers/devise.rb (hunk 1 above)
Comment (phyrog) [inline config/initializers/devise.rb]: Oh ok

File: db/migrate/20160902111740_initial_schema.rb (hunk 2 above)
Comment (phyrog) [inline db/migrate/20160902111740_initial_schema.rb]: This PR is fine I think

Comment (eugenk) [inline db/migrate/20160902111740_initial_schema.rb]: Done.

File: config/initializers/devise.rb [hunk 3]
```diff
@@ -2,12 +2,21 @@
...
   # using other encryptors, it sets how many times you want the password
   # re-encrypted.
   #
   # Limiting the stretches to just one in testing will increase the performance
   # of your test suite dramatically. However, it is STRONGLY RECOMMENDED to not
-  # use a value less than 15 in other environments.
-  config.stretches = Rails.env.test? ? 1 : 15
+  # use a value less than 10 in other environments.
+  config.stretches = Rails.env.test? ? 1 : 13
```
Comment (ebolloff) [inline config/initializers/devise.rb]: `10` or `13`? Edit comment.
